tests/slurm_plugin/test_clustermgtd.py

```diff
@@ -950,16 +950,15 @@ def test_handle_unhealthy_dynamic_nodes(
             [
                 SlurmNode("queue1-dy-c5xlarge-1", "ip-1", "hostname", "IDLE+CLOUD", "queue1"),
                 SlurmNode("queue1-dy-c5xlarge-2", "ip-2", "hostname", "POWERING_DOWN", "queue1"),
-                SlurmNode("queue1-dy-c5xlarge-3", "ip-3", "hostname", "IDLE+CLOUD+POWER_DOWN", "queue1"),
-                SlurmNode("queue1-dy-c5xlarge-4", "ip-4", "hostname", "IDLE+CLOUD+POWERED_", "queue1"),
+                SlurmNode("queue1-dy-c5xlarge-3", "ip-3", "hostname", "IDLE+CLOUD+POWER", "queue1"),
+                SlurmNode("queue1-dy-c5xlarge-4", "ip-4", "hostname", "IDLE+CLOUD+POWER_", "queue1"),
                 SlurmNode(
```

Comment:
For
`SlurmNode("queue1-dy-c5xlarge-3", "ip-3", "hostname", "IDLE+CLOUD+POWER_DOWN", "queue1"),`, I think we can leave it there instead of changing "IDLE+CLOUD+POWER_DOWN" to be ""IDLE+CLOUD+POWER", and remove it from the list of expected_powering_down_nodes.
For `SlurmNode("queue1-dy-c5xlarge-4", "ip-4", "hostname", "IDLE+CLOUD+POWERED_", "queue1")`, let's change it to `SlurmNode("queue1-dy-c5xlarge-4", "ip-4", "hostname", "IDLE+CLOUD+POWERED_DOWN", "queue1")`, and leave it in the list.
For test case `SlurmNode("queue1-dy-c5xlarge-7", "ip-7", "hostname", "IDLE+CLOUD+POWER_DOWN", "queue1")`, let's not delete it, but remove it from the list.
Let's add two more test cases here:
```
SlurmNode("queue1-st-c5xlarge-8", "ip-8", "hostname", "IDLE+CLOUD+DRAIN+POWER_DOWN", "queue1"),
SlurmNode("queue1-st-c5xlarge-9", "ip-9", "hostname", "IDLE+CLOUD+POWERING_DOWN", "queue1"),
```
They should also be added to the list.
In the code, let's also add:
cluster_manager._static_nodes_in_replacement = ["queue1-st-c5xlarge-6"] after `cluster_manager = ClusterManager(mock_sync_config)`, then we can remove `queue1-st-c5xlarge-6` from the list